addons/stock_account/wizard/stock_quantity_history.py

```diff
@@ -11,7 +12,8 @@ def open_at_date(self):
         if active_model == 'stock.valuation.layer':
             action = self.env.ref('stock_account.stock_valuation_layer_action').read()[0]
             action['domain'] = [('create_date', '<=', self.inventory_datetime), ('product_id.type', '=', 'product')]
-            action['display_name'] = str(self.inventory_datetime)
+            user_tz = self.env.user.tz or 'UTC'
```

Comment:
Use format_date in tool.misc.py

Reply:
@fmdl Thanks! I was pretty sure that there is a simpler/cleaner way :slightly_smiling_face: Done :+1: